tests/osautoinst/test_running.pm

```diff
@@ -13,7 +13,6 @@ sub run ($self) {
     if (get_var('FULL_MM_TEST')) {
         # we can't upload logs if the multimachine OVS bridge in the SUT has the same IP as the openQA-worker host
         script_run 'ip a del 10.0.2.2/15 dev br1'; # This may fail in case this IP is not actually set on the bridge
-        $self->upload_mm_logs();
```

Comment:
There might have been a reason why this was called during the normal test and not only in the post_fail_hook.

Reply:
This is still true, look at the next line, `upload_openqa_logs` should handle that now as the log uploading was added to it recently.